__tests__/test-worktree.js

```
@@ -0,0 +1,29 @@
+/* eslint-env node, browser, jasmine */
+import * as path from 'path'
+
+import { statusMatrix, add } from 'isomorphic-git'
```

ESLint

[error] 'add' is defined but never used.

(no-unused-vars)